app/controllers/api/v0/articles_controller.rb

```diff
@@ -11,6 +11,11 @@ class ArticlesController < ApiController
 
       skip_before_action :verify_authenticity_token, only: %i[create update]
 
+      wrap_parameters :article, include: %i[
+        title body_markdown published series main_image
+        canonical_url description tags
+      ]
```

Comment:
I just realized that this list is repeated down in `article_params` and it doesn't include `organization_id`